scripts/eks/appsignals/sample-app/nutrition-service-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nutrition-service-nodejs
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: nutrition-service-nodejs
+  template:
+    metadata:
+      labels:
+        io.kompose.service: nutrition-service-nodejs
+      annotations:
+        instrumentation.opentelemetry.io/inject-nodejs: 'true'
+    spec:
+      serviceAccountName: visits-service-account
+      containers:
+        - image: 111122223333.dkr.ecr.us-west-2.amazonaws.com/nodejs-petclinic-nutrition-service:latest
+          name: nutrition-service-nodejs
+          env:
+            - name: EUREKA_SERVER_URL
+              value: discovery-server
+            - name: MONGO_URI
+              value: ${DOCDB_URI}
```

Comment:
Do we have another example of where we are pulling in TF variables into k8's manifests?

Reply:
No, you can follow what Python does. We launched a database instance inside the cluster and connect to it.